specs/VSMAnalyticsRerunPipelineSatge.spec

```diff
@@ -0,0 +1,62 @@
+VSMAnalyticsRerunPipelineSatge
```

Comment:
Correct spelling - `VSMAnalyticsRerunPipelineStage.spec`. Correct the file name also